systemtest/containers.go

```diff
@@ -361,14 +370,27 @@ func (c *ElasticAgentContainer) Start() error {
 	defer cancel()
 
 	// Update request from user-definable fields.
-	if c.FleetEnrollmentToken != "" {
+	if c.FleetServer {
+		c.request.Env["FLEET_SERVER_ENABLE"] = "1"
+		c.request.Env["FLEET_SERVER_HOST"] = "0.0.0.0"
```

Comment:
Shouldn't be necessary anymore afaik.

Reply:
I just tried removing FLEET_SERVER_HOST and FLEET_SERVER_PORT (and tried just removing each individually), and the server never became accessible. In the elastic-agent log I see "State changed to CONFIG: Re-configuring". Maybe a bug masked by overriding the host/port?

Reply:
I'm going to merge as it is and come back to this, so we can get our system tests running again

Reply:
Yes sure; thanks for trying it out.